### DISCUSSION TITLE
How does the ontology capture survey effort?

### DISCUSSION THREAD

**Discussion by kheerand** [ternaustralia/ontology_tern#82] [General]
Below is a snippet from an email

> My understanding is that survey effort is the amount of time and effort that resulted in a record, either a positive (presence) or absence (negative). If a great deal of effort has been expended and the result is absence, then there is more confidence that the absence is true; conversely, a great deal of effort that results in a presence may indicate that the species is rare (or at least, rarely encountered). For example, if I visit an area and record a species with little effort then it’s probably common, but if it takes me a long time and a lot of effort to find it, then it’s probably rare.
> 
> Thinking through this more, there are two components of effort – time and sampling intensity. If I have many samples (observations) from an area and only one sample is of a target species, then it’s probably rare; many samples indicates that it’s probably common. Time can then be used as a surrogate or equivalent of samples – much time spent means many (potential) observations, so a long time to record a species indicates potential rarity.

While it is clear that the survey method is captured, is it currently capturing survey effort?  If not, how would it capture it?

**Comment by miekeGR** [comment]
Nice question @kheerand 
I am going to take a stab at this to continue this conversation. I was thinking the duration and frequency of tern:SiteVisit and the number of tern:Sampling events from a tern:Site would give you an idea of both time and intensity. 
Additionally to time and intensity, I see benefit in declaring the target of the survey. I am interested in how others think that might be articulated.

**Reply by edmondchuc** [reply]
Is it enough to annotate `tern:SiteVisit` instances with `tern:Attribute` to capture the sampling effort and sampling intensity? If you want to annotate site visits to filter based on sampling intensity, then you can probably do something like the below.

```turtle
ex:site-1 a tern:Site .

ex:site-1-visit-1 a tern:SiteVisit ;
    # ...
    tern:hasAttribute [
        tern:attribute <https://linked.data.gov.au/def/bdr-cv/sampling-intensity> ;
        tern:hasSimpleValue <https://linked.data.gov.au/def/bdr-cv/sampling-intensity/high> ;
        tern:hasValue [
            rdf:value <https://linked.data.gov.au/def/bdr-cv/sampling-intensity-values/high> ;
            tern:vocabulary <https://linked.data.gov.au/def/bdr-cv/sampling-intensity-values> ;
        ] ;
    ] ;
```

Annotate each site visit with the sampling intensity defined in the BDR controlled vocabularies. These can be automated during the ingestion into the BDR system. For example, calculate the sampling intensity based on some algorithm (time vs number of sampling points) and classify the site visit with a set of sampling intensity classifications (high, medium, low, etc.).

Is this what you are asking for @kheerand?
